### ISSUE TITLE
Line too long: Long triple-quoted string

### ISSUE THREAD

**Issue by brettcannon** [psf/black#1617]
**Describe the bug** Adding an `=` within an f-string that is too long to fit on a line causes Black to undo the work it did to try and make the line fit within the column restrictions

**To Reproduce** Steps to reproduce the behavior:

1. Take this code:
```python
class Wraps:
    binary_inplace_op.__doc__ = f"""Implement the augmented arithmetic assignment `a {binary_op.operator} b`."""


class NoWrap:
    binary_inplace_op.__doc__ = f"""Implement the augmented arithmetic assignment `a {binary_op.operator}= b`."""
```
2. Run Black
3. Notice how the line in the `Wraps` class gets wrapped to try and make the line shorter, while in the `NoWrap` class the introduction of a `=` in the f-string undoes the wrapping.

**Expected behavior**
To see the line with the `=` in the f-string` still be wrapped since it is now even longer than before.

**Environment (please complete the following information):**

- Version: 19.10b0
- OS and Python version: CPython 3.8.3, macOS

**Does this bug also happen on master?**

[Yes](https://black.now.sh/?version=master&state=_Td6WFoAAATm1rRGAgAhARYAAAB0L-Wj4AFQAKRdAD2IimZxl1N_WlXnON2nzOlliRI9h4n4vpS3v9GXQgSzJMk9B31wOGEqJOYvSnwRDRZHc8DOe4PTyMGg9rQqmyFjnFskhUunACaNmVWMMYpHmzMZQWCPNkTlqxUJ8rv56IocDBojnbod5tVDYIxF-tncjzH2lFW-1iO6Oi9XSdMz580PWPNgs4jXXNDSGZoX0aFmMkqv2iU2XQrdxQTtzohoUtMyANQ5LuV_EXYiAAHAAdECAABL6bFyscRn-wIAAAAABFla).

**Comment by brettcannon** [comment]
@JelleZijlstra same results.

**Comment by ambv** [comment]
Yeah, this is not a bug. There's no point putting additional parentheses around things that won't fit anyway. If you split the string into two, you'll see correct behavior. It's a bug that `--experimental-string-processing` doesn't do this automatically for you though.

**Comment by bbugyi200** [comment]
@JelleZijlstra @ambv I know I'm late the party here, but note that `--experimental-string-processing` (e.g. https://github.com/psf/black/pull/1132) does not attempt to improve (or change at all) the way black handles triple-quoted strings.
